kernel/arch/dreamcast/hardware/maple/vmu.c

```diff
@@ -433,7 +574,7 @@ static int vmu_block_write_internal(maple_device_t * dev, uint16 blocknum, uint8
 
 /* Sometimes a flaky or stubborn card can be recovered by trying a couple
    of times... */
-int vmu_block_write(maple_device_t * dev, uint16 blocknum, uint8 *buffer) {
+int vmu_block_write(maple_device_t *dev, const uint16_t blocknum, const uint8_t *buffer) {
```

Comment:
Why is this `const`?
```suggestion
int vmu_block_write(maple_device_t *dev, uint16_t blocknum, const uint8_t *buffer) {
```

Reply:
Good call, there's no point in making a value arg like that const. Not sure wtf I did.